include/parakeet/macros.h

```diff
@@ -0,0 +1,9 @@
+/*
+	Copyright 2021 OpenJAUS, LLC (dba MechaSpin). Subject to the MIT license.
+*/
+
+#if defined(_WIN32)
+    #define DEPRECATED(func) __declspec(deprecated) func
```

Comment:
We probably want to namespace this macro to avoid potential conflicts; we've run into that problem with the openjaus code base. Something like PARAKEET_DEPRECATED or PRKT_DEPRECATED if we're okay with using PRKT as an abbreviation.

Reply:
Went with PARAKEET_DEPRECATED.